services/wfm/src/main/java/org/openkilda/wfm/topology/network/storm/bolt/speaker/command/SpeakerFlowWorkerCommand.java

```diff
@@ -0,0 +1,33 @@
+/* Copyright 2019 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.wfm.topology.network.storm.bolt.speaker.command;
+
+import org.openkilda.wfm.topology.network.storm.ICommand;
+import org.openkilda.wfm.topology.network.storm.bolt.speaker.SpeakerFlowWorker;
+
+import lombok.Getter;
+
+public abstract class SpeakerFlowWorkerCommand implements ICommand<SpeakerFlowWorker> {
```

Comment:
I don't like the idea to reuse OpenFlow terms (like Flow) outside Speaker. Please, reconsider the names.

Reply:
renamed with SpeakerRulesWorkerCommand here, and in the all places along network topology